tests/Generators.hs

```diff
@@ -0,0 +1,88 @@
+module Generators
+  ( -- * Times
+    time
+  , timeInRange
+
+    -- * Timeout options
+  , timingOptions
+  , TimeoutGenOptions (..)
+  , defaultTimeoutGenOptions
+
+    -- * Session data
+  , sessionData
+  ) where
+
+import Yesod.Session.Persist.Prelude
+
+import Yesod.Session.Persist
+
+import Test.Hspec.Hedgehog
+
+import Hedgehog.Gen qualified as Gen
+import Hedgehog.Range qualified as Range
```

Comment:
I don't think hedgehog's too scary but should we clear it with folks?

Reply:
I'm learning more about QuickCheck and thinking I may want to just switch it all over from Hedgehog, will see how that goes.

Reply:
Okay, I did switch everything over to QuickCheck. The uniformity is an improvement.